ICAFIX/ReApplyFixMultiRunPipeline.sh

```diff
@@ -501,10 +501,10 @@ main()
 	# Several additional necessary environment variables (e.g., ${FSL_FIX_CIFTIRW} and ${FSL_FIX_WBC})
 	# are set in ${FSL_FIXDIR}/settings.sh, which is sourced below for interpreted modes.
 	# Note that fix_3_clean.m *appends* ${FSL_FIX_CIFTIRW} to the matlab path (i.e., the ciftiopen.m, ciftisave.m functions).
-	# BUT the CIFTI I/O functions are also now included within '${HCPPIPEDIR}/global/matlab' as well.
-	# AND since 'addpath' *prepends* to the matlab path, the versions in '${HCPPIPEDIR}/global/matlab' will therefore take precedence!
+	# The pipelines CIFTI I/O functions have been moved to a subdirectory, and the SetUp... script selects what to use with HCPCIFTIRWDIR.
+	# AND since 'addpath' *prepends* to the matlab path, the versions in '${HCPPIPEDIR}/global/matlab' will therefore take precedence.
```

Comment:
Comment should be changed to:
```suggestion
	# AND since 'addpath' *prepends* to the matlab path, the versions in '${HCPCIFTIRWDIR}' will therefore take precedence.
```

Reply:
well it all depends on the order of the addpath commands...

Reply:
Well, `${HCPPIPEDIR}/global/matlab` will have precedence in the path, but it won't contain any CIFTI I/O functions, which is what is being referred to here.  But, I see the potential source of confusion regarding my suggestion, and have edited it appropriately.

Reply:
Since FSL_FIX_CIFTIRW isn't mentioned in the file, it would probably be better to say something like "take precedence over the fix settings file".

Reply:
That's fine with me for the line in question, although given the convoluted nature by which this all works together, I think it is still valuable to maintain the mentions of  `${FSL_FIX_CIFTIRW}` and `${FSL_FIX_WBC}` earlier in the comment.